### DISCUSSION TITLE
React PropTypes support in Next Js

### DISCUSSION THREAD

**Discussion by debabrata100** [vercel/next.js#12950] [Help] [answered]
Hello Guys,
When I use propTypes for components, I am not able to any warnings or errors. 
Does anyone know how to use PropTypes in a Next App.
Please help me out!

**Comment by rafaelalmeidatk** [comment, marked as answer]
Hello! You don't have to do anything special to use prop-types with Next, can you share your code so we can give a look at what may be wrong?

**Reply by debabrata100** [reply]
Hi @rafaelalmeidatk , my code as following snippet:

`import PropsTypes from 'prop-types';
export default function NewsList ({ newsList, error, onHideNews, onVote }) {
   return (<div>NewsList</div>);
}
NewsList.propsTypes = {
    newsList: PropsTypes.string,
    error: PropsTypes,
    onHideNews: PropsTypes.func,
    onVote: PropsTypes.func
}`

For **newsList**  I am passing an array, But I don't get any errors for to be a string.

**Reply by rafaelalmeidatk** [reply]
The correct property name is `propTypes`, in this example it is written `propsTypes` (notice the `s` in `props`)

**Reply by debabrata100** [reply]
Hi @rafaelalmeidatk 
I made the changes and its working fine.
Thanks, It was great help and Pardon for asking silly questions, it was my bad.
